Sources/WebDriver/Session.swift

```diff
@@ -308,31 +308,51 @@ public class Session {
         try webDriver.send(Requests.SessionWindow.Delete(session: id, name: name))
     }
 
-    public func size(window handle: String) throws -> (width: Int, height: Int) {
-        let response = try webDriver.send(Requests.SessionWindowSize.Get(session: id, windowHandle: handle))
+    public func size(window handle: String) throws -> (width: Double, height: Double) {
```

Comment:
Remove in favor of providing a way to get a Window instance, which then has the window-specific members:

```
public func window(handle: String) -> Window { .init(session: self, handle: handle) }
```

(same for other methods below)